src/config/config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+   HOST: "localhost",
+   USER: "postgres",
+   PASSWORD: "ubuntu",
+   DB: "phathom",
+   dialect: "postgres",
```

Comment:
This will not work in production, please consider making values as environment variables rather than hardcoding 'em

Reply:
on it